app/scripts/lib/channels/notifier.js

```diff
@@ -18,9 +18,10 @@ define(function (require, exports, module) {
   var EVENTS = {
     COMPLETE_RESET_PASSWORD_TAB_OPEN: 'fxaccounts:complete_reset_password_tab_open',
     DELETE: 'fxaccounts:delete',
+    DEVICE_NAME_CHANGE: 'fxaccounts:device_name_change',
     PROFILE_CHANGE: 'profile:change',
     SIGNED_IN: 'internal:signed_in',
-    SIGNED_OUT: 'internal:signed_out'
+    SIGNED_OUT: 'fxaccounts:logout'
```

Comment:
to clarify: 
Content Server should be sending this logout event but not receiving. we need this change but see below, we need to adjust `triggerAll`, see https://github.com/mozilla/fxa-content-server/pull/3274/files#r45998248